utils/phpstan-baseline/method.childReturnType.neon

```diff
@@ -137,13 +137,18 @@ parameters:
             count: 1
             path: ../../system/Images/Handlers/BaseHandler.php
 
+        -
+            message: '#^Return type \(CodeIgniter\\Images\\Handlers\\ImageMagickHandler\) of method CodeIgniter\\Images\\Handlers\\ImageMagickHandler\:\:_crop\(\) should be covariant with return type \(\$this\(CodeIgniter\\Images\\Handlers\\BaseHandler\)\) of method CodeIgniter\\Images\\Handlers\\BaseHandler\:\:_crop\(\)$#'
+            count: 1
+            path: ../../system/Images/Handlers/ImageMagickHandler.php
+
         -
             message: '#^Return type \(CodeIgniter\\Images\\Handlers\\ImageMagickHandler\) of method CodeIgniter\\Images\\Handlers\\ImageMagickHandler\:\:_resize\(\) should be covariant with return type \(\$this\(CodeIgniter\\Images\\Handlers\\BaseHandler\)\) of method CodeIgniter\\Images\\Handlers\\BaseHandler\:\:_resize\(\)$#'
             count: 1
             path: ../../system/Images/Handlers/ImageMagickHandler.php
 
         -
-            message: '#^Return type \(bool\|CodeIgniter\\Images\\Handlers\\ImageMagickHandler\) of method CodeIgniter\\Images\\Handlers\\ImageMagickHandler\:\:_crop\(\) should be covariant with return type \(\$this\(CodeIgniter\\Images\\Handlers\\BaseHandler\)\) of method CodeIgniter\\Images\\Handlers\\BaseHandler\:\:_crop\(\)$#'
+            message: '#^Return type \(mixed\) of method CodeIgniter\\Images\\Handlers\\ImageMagickHandler\:\:getVersion\(\) should be covariant with return type \(string\) of method CodeIgniter\\Images\\Handlers\\BaseHandler\:\:getVersion\(\)$#'
```

Comment:
Saw this change to `string|false` in above. Should we just return empty string when version cannot be determined?